packages/devextreme/testing/tests/DevExpress.ui.widgets.form/formLayoutManager.markup.tests.js

```diff
@@ -2688,6 +2688,40 @@ QUnit.module('Supported editors', () => {
         assert.deepEqual(layoutManager.option('layoutData.noRange'), [2, 6], 'data updated');
     });
 
+    test('RangeSlider should be rendered with passed name attributes for startName and endName inputs (T1256743)', function(assert) {
+        const layoutManager = $('#container').dxLayoutManager({
+            layoutData: {
+                range: [10, 20]
+            },
+            items: [{
+                dataField: 'CustomRange',
+                editorType: 'dxRangeSlider',
+                editorOptions: {
+                    startName: 'RangeStart',
+                    endName: 'RangeEnd'
+                }
+            }]
+        }).dxLayoutManager('instance');
+
+        assert.deepEqual(layoutManager.getEditor('CustomRange').option('startName'), 'RangeStart', 'startName specified correctly');
+        assert.deepEqual(layoutManager.getEditor('CustomRange').option('endName'), 'RangeEnd', 'endName specified correctly');
+    });
+
+    test('DateRangeBox should be rendered with specified name attributes for startName and endName inputs (T1256743)', function(assert) {
```

Comment:
Could you please make tests names more precise? What data and where should be used. Now I read names on these 3 tests and see the same scenario

Reply:
Adjusted https://github.com/DevExpress/DevExtreme/pull/29153/commits/ec2a8e9ad692e0518f3a86d07e64e9237ac36395